src/app/main/component/events/components/event-editor/components/create-event-information/create-event-information.component.spec.ts

```diff
@@ -14,6 +14,18 @@ import { ContentChange, QuillModule } from 'ngx-quill';
 import { Language } from '../../../../../../i18n/Language';
 import { CreateEventInformationComponent } from './create-event-information.component';
 
+class QuillMock {
+  selection: { index: number; length: number } | null = { index: 0, length: 0 };
+
+  getSelection() {
```

Comment:
Unused methods getSelection, setSelection

Reply:
they are being used in component and are mocked here